### PR TITLE
Support new operators

### PR DESCRIPTION
**Pull request by vladmos**
The unary `+` operator is implemented in Bazel in https://github.com/bazelbuild/bazel/pull/8890, bitwise operators are supported by https://github.com/bazelbuild/bazel/pull/8903. Buildifier should support them too.

### CODE REVIEW

File: build/testdata/052.golden [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Unary operators
+
+-foo
+
++foo
```
Comment (laurentlb) [inline build/testdata/052.golden]: add test for `+ + foo` or `- + foo`

Comment (vladmos) [reply build/testdata/052.golden]: Done, also added support for bitwise operators (in the same PR)

File: build/print.go [hunk 2]
```diff
@@ -334,6 +334,10 @@ var opPrec = map[string]int{
 	"//":     precMultiply,
 	"%":      precMultiply,
 	"|":      precMultiply,
+	"&":      precMultiply,
+	"^":      precMultiply,
+	"<<":     precMultiply,
+	">>":     precMultiply,
```
Comment (laurentlb) [inline build/print.go]: This doens't seem to match 
https://docs.python.org/3/reference/expressions.html#operator-precedence or https://github.com/google/starlark-go/blob/master/doc/spec.md#binary-operators

Comment (vladmos) [reply build/print.go]: Fixed.

**Pull request merged**